Add sbgECom driver documentation

tolesam opened the pull request:
PX4-Autpilot corresponding PR: 
- https://github.com/PX4/PX4-Autopilot/pull/24137

hamishwillee commented on en/modules/modules_driver_ins.md:
```diff
@@ -1,7 +1,65 @@
 # Modules Reference: Ins (Driver)
```
These docs need to be included in your driver source code, and then will automatically be generated -  for example see https://docs.px4.io/main/en/modules/modules_driver_ins.html - the vectornav docs are created from this source: https://github.com/PX4/PX4-Autopilot/blob/main/src/drivers/ins/vectornav/VectorNav.cpp#L828

hamishwillee replied:
I.e. please remove this for now.

tolesam replied:
done

hamishwillee commented on en/advanced_config/parameter_reference.md:
```diff
@@ -30048,6 +30048,42 @@ Reboot | minValue | maxValue | increment | default | unit
 --- | --- | --- | --- | --- | ---
 &check; |  |  |  | 500000 |  
 
+### SBG_BAUDRATE (`INT32`) {#SBG_BAUDRATE}
```
The parameter docs are autogenerated from source. So when your driver PR is merged you'll get this stuff appearing automatically. Please remove this for now.

tolesam replied:
done

hamishwillee commented on en/sensor/sbgecom.md:
```diff
@@ -0,0 +1,148 @@
+
+[SBG-Systems](https://www.sbg-systems.com/) designs, manufactures, and support an extensive range of state-of-the-art inertial sensors such as Inertial Measurement Units (IMU), Attitude and Heading Reference Systems (AHRS), Inertial Navigation Systems with embedded GNSS (INS/GNSS) …
+
+![Ellipse](../../assets/hardware/sensors/inertial/ellipse-inertial-navigation-system.png)
```
Can you run prettier on this file please. Your repo is rejecting my fixes.

tolesam replied:
done
I've given you write access on the repo

hamishwillee replied:
Thank you!

hamishwillee commented:
1. Can you please provide the PR that this went in with? 
2. Can you add a release note (i.e. https://docs.px4.io/main/en/releases/main.html with this PR linked)
3. I'd guess the PR merged, but the docs for the module are not being generated? Or do they not exist yet?

hamishwillee commented:
@tolesam Ping!

tolesam commented:
@hamishwillee   
Thank you for your review.  
To answer your latest comments :
- I've added the PX4-Autopilot PR is this PR's description
- I've added it in the release note
- The PX4-Autopilot PR has not been merged yet, it's still under review

hamishwillee commented on en/releases/main.md:
```diff
@@ -55,7 +55,7 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 ### Sensors
 
-- TBD
+- Add sbgECom INS driver ([PX4-Autopilot#24137](https://github.com/PX4/PX4-Autopilot/pull/24137))
```
```suggestion
- Add [sbgECom INS driver](../sensor/sbgecom.md) ([PX4-Autopilot#24137](https://github.com/PX4/PX4-Autopilot/pull/24137))
```

tolesam commented:
> Thanks. I've done a minor structural update. We'll do another quick check once the associated PR goes in.

Thanks, I'll ping you as soon as the code PR is merged. I hope it will move ahead.

hamishwillee commented:
Hi @tolesam 
We moved the sources for docs to PX4-Autopilot in the /docs/ directory. This will therefore have to be copied across. Suggest we wait until the PR merges, though you can now move the docs changes into or alongside https://github.com/PX4/PX4-Autopilot/pull/24137 if you want.

Closing this version, in preparation for archiving the repo.
